### PR TITLE
96 update admin documentation

### PR DESCRIPTION
**Pull request by nichlaes**

### CODE REVIEW

File: docs/admin/client/auth.md [hunk 1]
```diff
@@ -102,4 +102,4 @@ DTaaS application URL: https://foo.com/bar
 Gitlab instance URL: https://foo.gitlab.com
 Callback URL: https://foo.com/bar/Library
 Logout URL: https://foo.com/bar
-```
+```
```
Comment (prasadtalasila) [inline docs/admin/client/auth.md:105]: Is there a change here?

File: docs/admin/guides/add_user.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# Add a new user
+
+This page will guide you on, how to add more users to the DTaas. Please do the following: 
+
+!!! important
+    Make sure to replace **<username\>** and **<port\>** 
+    Select a port that is not already being used by the system.
+
```
Comment (prasadtalasila) [inline docs/admin/guides/add_user.md:9]: Please add "Add the new user on Gitlab instance" as first step.

File: docs/admin/guides/hosting_site_without_https.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Hosting site without https
+
+The default setup is http. In order to host your site without https, please also see the [reverse proxy section on the overview page](../overview.md#reverse-proxy).
```
Comment (prasadtalasila) [inline docs/admin/guides/hosting_site_without_https.md]: In the default setup, the https connection is provided by the reverse proxy. The DTaaS application runs in http mode. So removing the reverse proxy removes the https mode.

File: docs/admin/guides/add_user.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    ...
+    <username>:
+      loadBalancer:
+        servers:
+          - url: 'http://localhost:<port>'
+```
+
+## 4. Access the new user
+
+Now the user should be accessible under `<yourdomain.com>/<username>` with the chosen credentials.
```
Comment (prasadtalasila) [inline docs/admin/guides/add_user.md:78]: Log into the DTaaS application as new user

File: docs/admin/guides/hosting_site_without_https.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# Hosting site without https
+
+The default setup is http. In order to host your site without https,
```
Comment (prasadtalasila) [inline docs/admin/guides/hosting_site_without_https.md]: The https is provided by the reverse proxy. The default installation of DTaaS runs in http mode.

File: docs/admin/guides/link_service.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+# Link services to local ports
+
+To link a port from the service machine (server2) to the local port on the user workspace (server1). You can make a ssh link between these two.
```
Comment (prasadtalasila) [inline docs/admin/guides/link_service.md:3]: please remove "(server1)"

File: docs/admin/guides/link_service.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
+# Link services to local ports
+
```
Comment (prasadtalasila) [inline docs/admin/guides/link_service.md:2]: Please add the requirements:
1. User needs to have an account on server2.
2. SSH server must be running on server2

File: mkdocs.yml [hunk 8]
```diff
@@ -30,6 +30,12 @@ nav:
         - website: admin/client/CLIENT.md
         - library microservice: admin/servers/lib/LIB-MS.md
       - Services: admin/services.md
+      - Guides:
```
Comment (prasadtalasila) [inline mkdocs.yml:33]: please add these to "mkdocs-github.yml" as well

File: docs/admin/client/auth.md (hunk 1 above)
Comment (nichlaes) [reply docs/admin/client/auth.md:105]: Maybe its because of the line ending

**Comment by prasadtalasila** [comment]
Please include scenario in issue #88 as well. All the required information is on that issue page. Please try the instructions to make sure that they work.

**Pull request closed without merging**